# Regression with Keras

Import the libraries and importing keras model amd layers

In [1]:
import pandas as pd
import numpy as np
import keras
from keras.models import Sequential
from keras.layers import Dense
from sklearn.model_selection import train_test_split

In [2]:
data=pd.read_csv("concrete_data.csv")
data.head()

Cement  Blast Furnace Slag  Fly Ash  Water  Superplasticizer  \
0   540.0                 0.0      0.0  162.0               2.5   
1   540.0                 0.0      0.0  162.0               2.5   
2   332.5               142.5      0.0  228.0               0.0   
3   332.5               142.5      0.0  228.0               0.0   
4   198.6               132.4      0.0  192.0               0.0   

   Coarse Aggregate  Fine Aggregate  Age  Strength  
0            1040.0           676.0   28     79.99  
1            1055.0           676.0   28     61.89  
2             932.0           594.0  270     40.27  
3             932.0           594.0  365     41.05  
4             978.4           825.5  360     44.30

In [3]:
data.shape

(1030, 9)

In [4]:
data.describe()

Cement  Blast Furnace Slag      Fly Ash        Water  \
count  1030.000000         1030.000000  1030.000000  1030.000000   
mean    281.167864           73.895825    54.188350   181.567282   
std     104.506364           86.279342    63.997004    21.354219   
min     102.000000            0.000000     0.000000   121.800000   
25%     192.375000            0.000000     0.000000   164.900000   
50%     272.900000           22.000000     0.000000   185.000000   
75%     350.000000          142.950000   118.300000   192.000000   
max     540.000000          359.400000   200.100000   247.000000   

       Superplasticizer  Coarse Aggregate  Fine Aggregate          Age  \
count       1030.000000       1030.000000     1030.000000  1030.000000   
mean           6.204660        972.918932      773.580485    45.662136   
std            5.973841         77.753954       80.175980    63.169912   
min            0.000000        801.000000      594.000000     1.000000   
25%            0.000000        932.000000      730.950000     7.000000   
50%            6.400000        968.000000      779.500000    28.000000   
75%           10.200000       1029.400000      824.000000    56.000000   
max           32.200000       1145.000000      992.600000   365.000000   

          Strength  
count  1030.000000  
mean     35.817961  
std      16.705742  
min       2.330000  
25%      23.710000  
50%      34.445000  
75%      46.135000  
max      82.600000

So the first concrete sample has 540 cubic meter of cement, 0 cubic meter of blast furnace slag, 0 cubic meter of fly ash, 162 cubic meter of water, 2.5 cubic meter of superplaticizer, 1040 cubic meter of coarse aggregate, 676 cubic meter of fine aggregate. Such a concrete mix which is 28 days old, has a compressive strength of 79.99 MPa. 


In [5]:
data.isnull().sum()

Cement                0
Blast Furnace Slag    0
Fly Ash               0
Water                 0
Superplasticizer      0
Coarse Aggregate      0
Fine Aggregate        0
Age                   0
Strength              0
dtype: int64

#### Split data into predictors and target

In [6]:
predictor=data.drop(columns='Strength')
target=data['Strength']

In [7]:
predictor_norm=(predictor-predictor.mean())/predictor.std()
predictor_norm.head()

Cement  Blast Furnace Slag   Fly Ash     Water  Superplasticizer  \
0  2.476712           -0.856472 -0.846733 -0.916319         -0.620147   
1  2.476712           -0.856472 -0.846733 -0.916319         -0.620147   
2  0.491187            0.795140 -0.846733  2.174405         -1.038638   
3  0.491187            0.795140 -0.846733  2.174405         -1.038638   
4 -0.790075            0.678079 -0.846733  0.488555         -1.038638   

   Coarse Aggregate  Fine Aggregate       Age  
0          0.862735       -1.217079 -0.279597  
1          1.055651       -1.217079 -0.279597  
2         -0.526262       -2.239829  3.551340  
3         -0.526262       -2.239829  5.055221  
4          0.070492        0.647569  4.976069

In [8]:
X_train,X_test,Y_train,Y_test=train_test_split(predictor,target,test_size=0.3)

Let's do a quick sanity check of the predictors and the target dataframes.

#### Normalizing Data

In [9]:
n_cols=predictor.shape[1] 

## Build a Neural Network

In [11]:
def regression_model():
    
    # create model
    model=Sequential()
    model.add(Dense(10,activation='relu',input_shape=(n_cols,)))
    model.add(Dense(10,activation='relu'))
    model.add(Dense(10,activation='relu'))
    model.add(Dense(1))
    
    model.compile(optimizer='adam',loss='mean_squared_error')
    return model

The above function create a model that has three hidden layers, each of 5 hidden units.

## Train and Test the Network  

In [12]:
model=regression_model()

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


In [13]:
model.fit(X_train,Y_train,epochs=50)

Epoch 1/50
23/23 ━━━━━━━━━━━━━━━━━━━━ 2s 3ms/step - loss: 1992.3743
Epoch 2/50
23/23 ━━━━━━━━━━━━━━━━━━━━ 0s 2ms/step - loss: 425.0429 
Epoch 3/50
23/23 ━━━━━━━━━━━━━━━━━━━━ 0s 2ms/step - loss: 209.6944 
Epoch 4/50
23/23 ━━━━━━━━━━━━━━━━━━━━ 0s 2ms/step - loss: 208.8340 
Epoch 5/50
23/23 ━━━━━━━━━━━━━━━━━━━━ 0s 2ms/step - loss: 210.8364 
Epoch 6/50
23/23 ━━━━━━━━━━━━━━━━━━━━ 0s 2ms/step - loss: 194.8972 
Epoch 7/50
23/23 ━━━━━━━━━━━━━━━━━━━━ 0s 2ms/step - loss: 164.4822 
Epoch 8/50
23/23 ━━━━━━━━━━━━━━━━━━━━ 0s 2ms/step - loss: 171.8871 
Epoch 9/50
23/23 ━━━━━━━━━━━━━━━━━━━━ 0s 2ms/step - loss: 168.9374 
Epoch 10/50
23/23 ━━━━━━━━━━━━━━━━━━━━ 0s 2ms/step - loss: 171.0832 
Epoch 11/50
23/23 ━━━━━━━━━━━━━━━━━━━━ 0s 2ms/step - loss: 179.3734  
Epoch 12/50
23/23 ━━━━━━━━━━━━━━━━━━━━ 0s 2ms/step - loss: 170.7940 
Epoch 13/50
23/23 ━━━━━━━━━━━━━━━━━━━━ 0s 2ms/step - loss: 172.6313 
Epoch 14/50
23/23 ━━━━━━━━━━━━━━━━━━━━ 0s 2ms/step - loss: 150.2557 
Epoch 15/50
23/23 ━━━━━━━━━━━━━━━━━━━━ 0s 

In [14]:
predictions=model.predict(X_test)

10/10 ━━━━━━━━━━━━━━━━━━━━ 0s 8ms/step


In [15]:
predictions
# model.compile(optimizer='adam',loss='mean_squared_error')

array([[58.73176   ],
       [30.461832  ],
       [35.97067   ],
       [21.197561  ],
       [16.923752  ],
       [28.282816  ],
       [18.851711  ],
       [17.740343  ],
       [22.051956  ],
       [40.120865  ],
       [22.875114  ],
       [22.01788   ],
       [29.003304  ],
       [60.87757   ],
       [51.439857  ],
       [53.61368   ],
       [51.190952  ],
       [47.144352  ],
       [48.296867  ],
       [46.107903  ],
       [23.030632  ],
       [18.26268   ],
       [24.179514  ],
       [47.94485   ],
       [14.501095  ],
       [25.724598  ],
       [16.968288  ],
       [49.19458   ],
       [40.04694   ],
       [38.759026  ],
       [55.12574   ],
       [26.336615  ],
       [21.070515  ],
       [21.887564  ],
       [49.467125  ],
       [39.538383  ],
       [44.38529   ],
       [18.153824  ],
       [14.7019825 ],
       [23.139776  ],
       [71.34497   ],
       [32.918373  ],
       [38.768196  ],
       [44.21467   ],
       [55.598763  ],
       [46

In [16]:
n_cols = predictor.shape[1] # number of predictors
mean_squared_errors = []
from sklearn.metrics import mean_squared_error

for i in range(50):
    x_train, x_test, y_train, y_test = train_test_split(predictor, target, test_size=0.3)
    model = regression_model()
    model.fit(x_train, y_train, validation_data=(x_test, y_test), epochs=100, verbose=2)
    
    predictions=model.predict(X_test)
    mse = mean_squared_error(Y_test, predictions)
    mean_squared_errors.append(mse)
    



Epoch 1/100


C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 2s - 80ms/step - loss: 26448.9453 - val_loss: 14182.1475
Epoch 2/100
23/23 - 0s - 7ms/step - loss: 9094.8994 - val_loss: 5275.9165
Epoch 3/100
23/23 - 0s - 5ms/step - loss: 4024.9070 - val_loss: 2868.3479
Epoch 4/100
23/23 - 0s - 5ms/step - loss: 2279.2390 - val_loss: 1762.4739
Epoch 5/100
23/23 - 0s - 5ms/step - loss: 1397.5723 - val_loss: 1149.8301
Epoch 6/100
23/23 - 0s - 5ms/step - loss: 938.6027 - val_loss: 781.7204
Epoch 7/100
23/23 - 0s - 5ms/step - loss: 681.3923 - val_loss: 614.8434
Epoch 8/100
23/23 - 0s - 5ms/step - loss: 565.0300 - val_loss: 537.3608
Epoch 9/100
23/23 - 0s - 6ms/step - loss: 500.0534 - val_loss: 489.0579
Epoch 10/100
23/23 - 0s - 6ms/step - loss: 457.1810 - val_loss: 447.4451
Epoch 11/100
23/23 - 0s - 6ms/step - loss: 419.2731 - val_loss: 412.7209
Epoch 12/100
23/23 - 0s - 5ms/step - loss: 388.1376 - val_loss: 383.3922
Epoch 13/100
23/23 - 0s - 5ms/step - loss: 358.1991 - val_loss: 358.3255
Epoch 14/100
23/23 - 0s - 6ms/step - loss: 333.1131 - val_l

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 2s - 82ms/step - loss: 8744.3076 - val_loss: 995.4125
Epoch 2/100
23/23 - 0s - 5ms/step - loss: 1104.4596 - val_loss: 575.5016
Epoch 3/100
23/23 - 0s - 5ms/step - loss: 615.0167 - val_loss: 456.1599
Epoch 4/100
23/23 - 0s - 5ms/step - loss: 515.4215 - val_loss: 401.9942
Epoch 5/100
23/23 - 0s - 5ms/step - loss: 450.0685 - val_loss: 355.7427
Epoch 6/100
23/23 - 0s - 5ms/step - loss: 406.5126 - val_loss: 325.5708
Epoch 7/100
23/23 - 0s - 5ms/step - loss: 371.3656 - val_loss: 298.8663
Epoch 8/100
23/23 - 0s - 5ms/step - loss: 342.2063 - val_loss: 282.4792
Epoch 9/100
23/23 - 0s - 5ms/step - loss: 320.3103 - val_loss: 268.0540
Epoch 10/100
23/23 - 0s - 5ms/step - loss: 300.6036 - val_loss: 254.9690
Epoch 11/100
23/23 - 0s - 4ms/step - loss: 285.8588 - val_loss: 244.2049
Epoch 12/100
23/23 - 0s - 5ms/step - loss: 271.0843 - val_loss: 234.3359
Epoch 13/100
23/23 - 0s - 5ms/step - loss: 256.9818 - val_loss: 225.5005
Epoch 14/100
23/23 - 0s - 5ms/step - loss: 243.8750 - val_loss: 215.9

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 2s - 73ms/step - loss: 435207.1562 - val_loss: 269467.0625
Epoch 2/100
23/23 - 0s - 5ms/step - loss: 192259.4219 - val_loss: 126321.4531
Epoch 3/100
23/23 - 0s - 5ms/step - loss: 90240.6172 - val_loss: 53089.4023
Epoch 4/100
23/23 - 0s - 5ms/step - loss: 29892.8379 - val_loss: 14560.9414
Epoch 5/100
23/23 - 0s - 4ms/step - loss: 9539.8535 - val_loss: 5038.3130
Epoch 6/100
23/23 - 0s - 5ms/step - loss: 2722.4006 - val_loss: 892.3593
Epoch 7/100
23/23 - 0s - 4ms/step - loss: 464.7007 - val_loss: 318.8051
Epoch 8/100
23/23 - 0s - 5ms/step - loss: 264.6040 - val_loss: 283.6188
Epoch 9/100
23/23 - 0s - 5ms/step - loss: 248.7042 - val_loss: 277.5314
Epoch 10/100
23/23 - 0s - 5ms/step - loss: 244.9635 - val_loss: 272.8031
Epoch 11/100
23/23 - 0s - 5ms/step - loss: 240.3256 - val_loss: 267.3795
Epoch 12/100
23/23 - 0s - 5ms/step - loss: 233.7039 - val_loss: 260.3793
Epoch 13/100
23/23 - 0s - 4ms/step - loss: 227.2438 - val_loss: 253.5406
Epoch 14/100
23/23 - 0s - 5ms/step - loss: 221.8

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 2s - 70ms/step - loss: 596.3611 - val_loss: 404.6092
Epoch 2/100
23/23 - 0s - 5ms/step - loss: 353.8167 - val_loss: 357.0764
Epoch 3/100
23/23 - 0s - 5ms/step - loss: 304.1740 - val_loss: 333.2633
Epoch 4/100
23/23 - 0s - 5ms/step - loss: 267.1638 - val_loss: 282.2035
Epoch 5/100
23/23 - 0s - 5ms/step - loss: 222.5651 - val_loss: 204.5246
Epoch 6/100
23/23 - 0s - 5ms/step - loss: 167.2310 - val_loss: 158.0140
Epoch 7/100
23/23 - 0s - 5ms/step - loss: 136.5472 - val_loss: 134.8650
Epoch 8/100
23/23 - 0s - 5ms/step - loss: 122.4419 - val_loss: 120.5700
Epoch 9/100
23/23 - 0s - 5ms/step - loss: 115.1908 - val_loss: 114.5842
Epoch 10/100
23/23 - 0s - 5ms/step - loss: 106.9076 - val_loss: 119.9621
Epoch 11/100
23/23 - 0s - 5ms/step - loss: 107.6423 - val_loss: 106.8457
Epoch 12/100
23/23 - 0s - 5ms/step - loss: 102.6521 - val_loss: 108.9041
Epoch 13/100
23/23 - 0s - 4ms/step - loss: 100.6676 - val_loss: 102.3951
Epoch 14/100
23/23 - 0s - 5ms/step - loss: 98.8243 - val_loss: 100.0556

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 2s - 70ms/step - loss: 1371.4248 - val_loss: 738.2938
Epoch 2/100
23/23 - 0s - 5ms/step - loss: 389.4622 - val_loss: 354.8906
Epoch 3/100
23/23 - 0s - 5ms/step - loss: 298.8929 - val_loss: 325.0930
Epoch 4/100
23/23 - 0s - 5ms/step - loss: 274.9342 - val_loss: 308.5386
Epoch 5/100
23/23 - 0s - 5ms/step - loss: 259.0519 - val_loss: 289.2985
Epoch 6/100
23/23 - 0s - 5ms/step - loss: 243.6276 - val_loss: 273.7651
Epoch 7/100
23/23 - 0s - 5ms/step - loss: 233.0571 - val_loss: 260.8727
Epoch 8/100
23/23 - 0s - 5ms/step - loss: 224.1073 - val_loss: 249.5856
Epoch 9/100
23/23 - 0s - 5ms/step - loss: 215.9142 - val_loss: 242.0086
Epoch 10/100
23/23 - 0s - 5ms/step - loss: 207.8664 - val_loss: 231.7778
Epoch 11/100
23/23 - 0s - 5ms/step - loss: 201.6408 - val_loss: 232.2670
Epoch 12/100
23/23 - 0s - 5ms/step - loss: 196.0488 - val_loss: 224.0835
Epoch 13/100
23/23 - 0s - 5ms/step - loss: 190.5435 - val_loss: 208.5194
Epoch 14/100
23/23 - 0s - 5ms/step - loss: 183.2205 - val_loss: 210.62

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 2s - 72ms/step - loss: 7055.3057 - val_loss: 752.8949
Epoch 2/100
23/23 - 0s - 5ms/step - loss: 391.8285 - val_loss: 355.2287
Epoch 3/100
23/23 - 0s - 5ms/step - loss: 300.8589 - val_loss: 250.5144
Epoch 4/100
23/23 - 0s - 5ms/step - loss: 251.9700 - val_loss: 236.5426
Epoch 5/100
23/23 - 0s - 4ms/step - loss: 239.1109 - val_loss: 228.0013
Epoch 6/100
23/23 - 0s - 4ms/step - loss: 229.8920 - val_loss: 222.6734
Epoch 7/100
23/23 - 0s - 5ms/step - loss: 223.4205 - val_loss: 219.9731
Epoch 8/100
23/23 - 0s - 4ms/step - loss: 217.7027 - val_loss: 214.7992
Epoch 9/100
23/23 - 0s - 4ms/step - loss: 214.0063 - val_loss: 211.8895
Epoch 10/100
23/23 - 0s - 4ms/step - loss: 211.1751 - val_loss: 209.0129
Epoch 11/100
23/23 - 0s - 4ms/step - loss: 209.6011 - val_loss: 206.2415
Epoch 12/100
23/23 - 0s - 4ms/step - loss: 205.2430 - val_loss: 203.7202
Epoch 13/100
23/23 - 0s - 5ms/step - loss: 203.0851 - val_loss: 201.1582
Epoch 14/100
23/23 - 0s - 4ms/step - loss: 201.0589 - val_loss: 199.40

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 2s - 70ms/step - loss: 9435.9102 - val_loss: 1660.1106
Epoch 2/100
23/23 - 0s - 5ms/step - loss: 783.8769 - val_loss: 423.4767
Epoch 3/100
23/23 - 0s - 5ms/step - loss: 412.4404 - val_loss: 412.7357
Epoch 4/100
23/23 - 0s - 5ms/step - loss: 366.1773 - val_loss: 362.0229
Epoch 5/100
23/23 - 0s - 5ms/step - loss: 319.5157 - val_loss: 303.9063
Epoch 6/100
23/23 - 0s - 5ms/step - loss: 273.7929 - val_loss: 279.1509
Epoch 7/100
23/23 - 0s - 5ms/step - loss: 255.0637 - val_loss: 264.5413
Epoch 8/100
23/23 - 0s - 5ms/step - loss: 239.7896 - val_loss: 263.4545
Epoch 9/100
23/23 - 0s - 5ms/step - loss: 230.7003 - val_loss: 243.7473
Epoch 10/100
23/23 - 0s - 5ms/step - loss: 211.3178 - val_loss: 220.0551
Epoch 11/100
23/23 - 0s - 5ms/step - loss: 199.7350 - val_loss: 209.7523
Epoch 12/100
23/23 - 0s - 5ms/step - loss: 191.0532 - val_loss: 203.2492
Epoch 13/100
23/23 - 0s - 4ms/step - loss: 183.7852 - val_loss: 191.8702
Epoch 14/100
23/23 - 0s - 5ms/step - loss: 177.6456 - val_loss: 189.9

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 2s - 70ms/step - loss: 54437.2148 - val_loss: 22913.2012
Epoch 2/100
23/23 - 0s - 5ms/step - loss: 11606.1465 - val_loss: 3496.3318
Epoch 3/100
23/23 - 0s - 5ms/step - loss: 1566.3396 - val_loss: 556.3489
Epoch 4/100
23/23 - 0s - 5ms/step - loss: 447.4616 - val_loss: 463.6493
Epoch 5/100
23/23 - 0s - 5ms/step - loss: 393.3481 - val_loss: 396.0005
Epoch 6/100
23/23 - 0s - 5ms/step - loss: 339.6949 - val_loss: 330.4714
Epoch 7/100
23/23 - 0s - 5ms/step - loss: 287.3070 - val_loss: 275.1299
Epoch 8/100
23/23 - 0s - 5ms/step - loss: 244.1048 - val_loss: 222.1036
Epoch 9/100
23/23 - 0s - 5ms/step - loss: 213.6364 - val_loss: 191.4771
Epoch 10/100
23/23 - 0s - 5ms/step - loss: 199.8471 - val_loss: 176.2831
Epoch 11/100
23/23 - 0s - 5ms/step - loss: 190.0657 - val_loss: 173.5449
Epoch 12/100
23/23 - 0s - 5ms/step - loss: 184.2841 - val_loss: 162.4234
Epoch 13/100
23/23 - 0s - 5ms/step - loss: 181.6908 - val_loss: 157.3897
Epoch 14/100
23/23 - 0s - 4ms/step - loss: 173.6406 - val_loss:

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 2s - 81ms/step - loss: 4642.9199 - val_loss: 1015.6384
Epoch 2/100
23/23 - 0s - 5ms/step - loss: 565.6594 - val_loss: 476.9677
Epoch 3/100
23/23 - 0s - 5ms/step - loss: 424.8783 - val_loss: 402.1811
Epoch 4/100
23/23 - 0s - 5ms/step - loss: 371.8610 - val_loss: 369.9065
Epoch 5/100
23/23 - 0s - 5ms/step - loss: 340.1238 - val_loss: 340.4242
Epoch 6/100
23/23 - 0s - 5ms/step - loss: 314.9117 - val_loss: 316.5430
Epoch 7/100
23/23 - 0s - 5ms/step - loss: 293.2815 - val_loss: 296.1548
Epoch 8/100
23/23 - 0s - 4ms/step - loss: 274.7848 - val_loss: 276.3860
Epoch 9/100
23/23 - 0s - 4ms/step - loss: 255.6850 - val_loss: 259.1198
Epoch 10/100
23/23 - 0s - 4ms/step - loss: 238.6294 - val_loss: 241.4214
Epoch 11/100
23/23 - 0s - 5ms/step - loss: 221.2378 - val_loss: 226.6888
Epoch 12/100
23/23 - 0s - 5ms/step - loss: 208.9515 - val_loss: 214.5310
Epoch 13/100
23/23 - 0s - 5ms/step - loss: 195.9602 - val_loss: 201.3790
Epoch 14/100
23/23 - 0s - 5ms/step - loss: 183.5902 - val_loss: 191.0

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 2s - 71ms/step - loss: 28909.5137 - val_loss: 7023.2793
Epoch 2/100
23/23 - 0s - 5ms/step - loss: 2092.1462 - val_loss: 374.6544
Epoch 3/100
23/23 - 0s - 5ms/step - loss: 332.5800 - val_loss: 331.8564
Epoch 4/100
23/23 - 0s - 5ms/step - loss: 298.1391 - val_loss: 304.0848
Epoch 5/100
23/23 - 0s - 5ms/step - loss: 264.8614 - val_loss: 300.4503
Epoch 6/100
23/23 - 0s - 5ms/step - loss: 252.9654 - val_loss: 283.9405
Epoch 7/100
23/23 - 0s - 5ms/step - loss: 242.6474 - val_loss: 271.5847
Epoch 8/100
23/23 - 0s - 5ms/step - loss: 233.2090 - val_loss: 260.8654
Epoch 9/100
23/23 - 0s - 5ms/step - loss: 225.1723 - val_loss: 253.3355
Epoch 10/100
23/23 - 0s - 4ms/step - loss: 217.7970 - val_loss: 245.4505
Epoch 11/100
23/23 - 0s - 4ms/step - loss: 210.7858 - val_loss: 234.5075
Epoch 12/100
23/23 - 0s - 5ms/step - loss: 204.7063 - val_loss: 231.7801
Epoch 13/100
23/23 - 0s - 5ms/step - loss: 200.1564 - val_loss: 226.4056
Epoch 14/100
23/23 - 0s - 5ms/step - loss: 195.5712 - val_loss: 220

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 2s - 69ms/step - loss: 705.5408 - val_loss: 360.9444
Epoch 2/100
23/23 - 0s - 5ms/step - loss: 384.0354 - val_loss: 360.2207
Epoch 3/100
23/23 - 0s - 5ms/step - loss: 365.9885 - val_loss: 339.5062
Epoch 4/100
23/23 - 0s - 5ms/step - loss: 355.3897 - val_loss: 329.3669
Epoch 5/100
23/23 - 0s - 5ms/step - loss: 345.4626 - val_loss: 320.1091
Epoch 6/100
23/23 - 0s - 5ms/step - loss: 336.1587 - val_loss: 311.2033
Epoch 7/100
23/23 - 0s - 5ms/step - loss: 327.6492 - val_loss: 302.2573
Epoch 8/100
23/23 - 0s - 5ms/step - loss: 320.2083 - val_loss: 295.7069
Epoch 9/100
23/23 - 0s - 5ms/step - loss: 313.1498 - val_loss: 290.0033
Epoch 10/100
23/23 - 0s - 5ms/step - loss: 309.3461 - val_loss: 285.6822
Epoch 11/100
23/23 - 0s - 5ms/step - loss: 301.3648 - val_loss: 279.3216
Epoch 12/100
23/23 - 0s - 5ms/step - loss: 296.3382 - val_loss: 273.1730
Epoch 13/100
23/23 - 0s - 5ms/step - loss: 290.1916 - val_loss: 273.9779
Epoch 14/100
23/23 - 0s - 5ms/step - loss: 288.5886 - val_loss: 265.925

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 2s - 71ms/step - loss: 2281.8296 - val_loss: 993.2725
Epoch 2/100
23/23 - 0s - 5ms/step - loss: 669.0258 - val_loss: 424.4809
Epoch 3/100
23/23 - 0s - 5ms/step - loss: 410.9947 - val_loss: 324.4023
Epoch 4/100
23/23 - 0s - 5ms/step - loss: 358.3605 - val_loss: 296.5417
Epoch 5/100
23/23 - 0s - 5ms/step - loss: 338.8512 - val_loss: 286.2962
Epoch 6/100
23/23 - 0s - 5ms/step - loss: 328.4798 - val_loss: 280.3669
Epoch 7/100
23/23 - 0s - 5ms/step - loss: 321.6312 - val_loss: 276.2933
Epoch 8/100
23/23 - 0s - 4ms/step - loss: 314.9310 - val_loss: 271.0607
Epoch 9/100
23/23 - 0s - 5ms/step - loss: 311.0665 - val_loss: 267.1661
Epoch 10/100
23/23 - 0s - 5ms/step - loss: 305.8552 - val_loss: 263.2898
Epoch 11/100
23/23 - 0s - 5ms/step - loss: 299.8868 - val_loss: 258.7968
Epoch 12/100
23/23 - 0s - 5ms/step - loss: 295.7349 - val_loss: 255.2167
Epoch 13/100
23/23 - 0s - 5ms/step - loss: 290.4525 - val_loss: 251.7818
Epoch 14/100
23/23 - 0s - 5ms/step - loss: 285.6505 - val_loss: 246.90

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 2s - 71ms/step - loss: 10203.5176 - val_loss: 3921.1577
Epoch 2/100
23/23 - 0s - 5ms/step - loss: 3245.5535 - val_loss: 2375.0288
Epoch 3/100
23/23 - 0s - 5ms/step - loss: 1629.9146 - val_loss: 1284.0165
Epoch 4/100
23/23 - 0s - 5ms/step - loss: 795.6323 - val_loss: 707.1897
Epoch 5/100
23/23 - 0s - 5ms/step - loss: 500.8466 - val_loss: 458.1861
Epoch 6/100
23/23 - 0s - 5ms/step - loss: 355.0319 - val_loss: 330.6148
Epoch 7/100
23/23 - 0s - 5ms/step - loss: 276.3773 - val_loss: 281.6401
Epoch 8/100
23/23 - 0s - 5ms/step - loss: 235.3645 - val_loss: 235.0080
Epoch 9/100
23/23 - 0s - 5ms/step - loss: 196.1362 - val_loss: 213.0792
Epoch 10/100
23/23 - 0s - 5ms/step - loss: 175.4609 - val_loss: 179.8680
Epoch 11/100
23/23 - 0s - 5ms/step - loss: 164.1596 - val_loss: 173.1164
Epoch 12/100
23/23 - 0s - 5ms/step - loss: 156.7702 - val_loss: 167.3470
Epoch 13/100
23/23 - 0s - 4ms/step - loss: 148.9189 - val_loss: 160.1829
Epoch 14/100
23/23 - 0s - 5ms/step - loss: 141.5849 - val_loss: 

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 2s - 71ms/step - loss: 15711.3721 - val_loss: 7299.5532
Epoch 2/100
23/23 - 0s - 4ms/step - loss: 4583.7998 - val_loss: 2163.6091
Epoch 3/100
23/23 - 0s - 4ms/step - loss: 1305.4370 - val_loss: 760.2144
Epoch 4/100
23/23 - 0s - 4ms/step - loss: 689.9606 - val_loss: 604.1694
Epoch 5/100
23/23 - 0s - 4ms/step - loss: 549.0319 - val_loss: 506.2576
Epoch 6/100
23/23 - 0s - 4ms/step - loss: 466.3724 - val_loss: 435.9418
Epoch 7/100
23/23 - 0s - 4ms/step - loss: 411.8915 - val_loss: 401.4949
Epoch 8/100
23/23 - 0s - 4ms/step - loss: 376.5166 - val_loss: 366.9801
Epoch 9/100
23/23 - 0s - 4ms/step - loss: 349.3343 - val_loss: 346.2411
Epoch 10/100
23/23 - 0s - 4ms/step - loss: 328.7335 - val_loss: 322.7259
Epoch 11/100
23/23 - 0s - 4ms/step - loss: 310.0118 - val_loss: 304.1261
Epoch 12/100
23/23 - 0s - 4ms/step - loss: 292.9062 - val_loss: 287.0385
Epoch 13/100
23/23 - 0s - 4ms/step - loss: 276.0677 - val_loss: 274.6221
Epoch 14/100
23/23 - 0s - 4ms/step - loss: 258.9245 - val_loss: 2

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 2s - 70ms/step - loss: 34236.5586 - val_loss: 4174.6362
Epoch 2/100
23/23 - 0s - 5ms/step - loss: 1292.3441 - val_loss: 985.5026
Epoch 3/100
23/23 - 0s - 5ms/step - loss: 759.8221 - val_loss: 534.0325
Epoch 4/100
23/23 - 0s - 5ms/step - loss: 510.4334 - val_loss: 492.6239
Epoch 5/100
23/23 - 0s - 4ms/step - loss: 466.0376 - val_loss: 443.9783
Epoch 6/100
23/23 - 0s - 5ms/step - loss: 436.9786 - val_loss: 421.1937
Epoch 7/100
23/23 - 0s - 4ms/step - loss: 413.4852 - val_loss: 401.2079
Epoch 8/100
23/23 - 0s - 5ms/step - loss: 393.4327 - val_loss: 385.5639
Epoch 9/100
23/23 - 0s - 4ms/step - loss: 374.5597 - val_loss: 366.9375
Epoch 10/100
23/23 - 0s - 5ms/step - loss: 357.6066 - val_loss: 351.2292
Epoch 11/100
23/23 - 0s - 5ms/step - loss: 344.2284 - val_loss: 336.7876
Epoch 12/100
23/23 - 0s - 5ms/step - loss: 331.7950 - val_loss: 325.3747
Epoch 13/100
23/23 - 0s - 5ms/step - loss: 316.2461 - val_loss: 311.4728
Epoch 14/100
23/23 - 0s - 5ms/step - loss: 304.0655 - val_loss: 302

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 2s - 82ms/step - loss: 5923.9946 - val_loss: 2808.4116
Epoch 2/100
23/23 - 0s - 5ms/step - loss: 1944.0521 - val_loss: 1192.4379
Epoch 3/100
23/23 - 0s - 5ms/step - loss: 937.9973 - val_loss: 599.8514
Epoch 4/100
23/23 - 0s - 5ms/step - loss: 485.5296 - val_loss: 310.5818
Epoch 5/100
23/23 - 0s - 5ms/step - loss: 301.1279 - val_loss: 227.7926
Epoch 6/100
23/23 - 0s - 5ms/step - loss: 257.9879 - val_loss: 218.5058
Epoch 7/100
23/23 - 0s - 5ms/step - loss: 248.5573 - val_loss: 212.8085
Epoch 8/100
23/23 - 0s - 5ms/step - loss: 240.9860 - val_loss: 208.4331
Epoch 9/100
23/23 - 0s - 4ms/step - loss: 235.3717 - val_loss: 205.2469
Epoch 10/100
23/23 - 0s - 5ms/step - loss: 231.3228 - val_loss: 202.6831
Epoch 11/100
23/23 - 0s - 5ms/step - loss: 228.5149 - val_loss: 200.2085
Epoch 12/100
23/23 - 0s - 5ms/step - loss: 225.4749 - val_loss: 198.1443
Epoch 13/100
23/23 - 0s - 5ms/step - loss: 223.3361 - val_loss: 195.7215
Epoch 14/100
23/23 - 0s - 5ms/step - loss: 221.0289 - val_loss: 193

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 2s - 71ms/step - loss: 144790.2500 - val_loss: 36209.5859
Epoch 2/100
23/23 - 0s - 5ms/step - loss: 12388.4092 - val_loss: 1770.0717
Epoch 3/100
23/23 - 0s - 5ms/step - loss: 2001.5078 - val_loss: 1856.7144
Epoch 4/100
23/23 - 0s - 5ms/step - loss: 1651.9397 - val_loss: 1522.6265
Epoch 5/100
23/23 - 0s - 5ms/step - loss: 1494.0599 - val_loss: 1436.6368
Epoch 6/100
23/23 - 0s - 5ms/step - loss: 1391.8807 - val_loss: 1348.8358
Epoch 7/100
23/23 - 0s - 5ms/step - loss: 1306.1610 - val_loss: 1267.8984
Epoch 8/100
23/23 - 0s - 5ms/step - loss: 1220.5145 - val_loss: 1197.3977
Epoch 9/100
23/23 - 0s - 5ms/step - loss: 1146.9526 - val_loss: 1128.7269
Epoch 10/100
23/23 - 0s - 5ms/step - loss: 1081.4229 - val_loss: 1065.4419
Epoch 11/100
23/23 - 0s - 5ms/step - loss: 1013.5367 - val_loss: 1004.0533
Epoch 12/100
23/23 - 0s - 4ms/step - loss: 953.2853 - val_loss: 949.2534
Epoch 13/100
23/23 - 0s - 5ms/step - loss: 901.7607 - val_loss: 897.9706
Epoch 14/100
23/23 - 0s - 5ms/step - loss: 85

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 2s - 71ms/step - loss: 6009.8496 - val_loss: 2568.4204
Epoch 2/100
23/23 - 0s - 5ms/step - loss: 1912.4733 - val_loss: 1486.5575
Epoch 3/100
23/23 - 0s - 5ms/step - loss: 1597.1744 - val_loss: 1473.8501
Epoch 4/100
23/23 - 0s - 5ms/step - loss: 1593.4695 - val_loss: 1472.5354
Epoch 5/100
23/23 - 0s - 5ms/step - loss: 1592.2244 - val_loss: 1471.3118
Epoch 6/100
23/23 - 0s - 5ms/step - loss: 1590.9410 - val_loss: 1470.0408
Epoch 7/100
23/23 - 0s - 5ms/step - loss: 1589.6040 - val_loss: 1468.7344
Epoch 8/100
23/23 - 0s - 5ms/step - loss: 1588.2356 - val_loss: 1467.3975
Epoch 9/100
23/23 - 0s - 5ms/step - loss: 1586.8386 - val_loss: 1466.0211
Epoch 10/100
23/23 - 0s - 5ms/step - loss: 1585.4105 - val_loss: 1464.6324
Epoch 11/100
23/23 - 0s - 5ms/step - loss: 1583.9657 - val_loss: 1463.2299
Epoch 12/100
23/23 - 0s - 5ms/step - loss: 1582.5062 - val_loss: 1461.8176
Epoch 13/100
23/23 - 0s - 5ms/step - loss: 1581.0345 - val_loss: 1460.3799
Epoch 14/100
23/23 - 0s - 5ms/step - loss: 15

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 2s - 70ms/step - loss: 799.2191 - val_loss: 348.3996
Epoch 2/100
23/23 - 0s - 5ms/step - loss: 252.7649 - val_loss: 328.2032
Epoch 3/100
23/23 - 0s - 5ms/step - loss: 239.9464 - val_loss: 319.8879
Epoch 4/100
23/23 - 0s - 5ms/step - loss: 230.5479 - val_loss: 306.6532
Epoch 5/100
23/23 - 0s - 5ms/step - loss: 224.6887 - val_loss: 297.7231
Epoch 6/100
23/23 - 0s - 5ms/step - loss: 218.4549 - val_loss: 290.9653
Epoch 7/100
23/23 - 0s - 5ms/step - loss: 211.4989 - val_loss: 277.2578
Epoch 8/100
23/23 - 0s - 5ms/step - loss: 203.7480 - val_loss: 270.3259
Epoch 9/100
23/23 - 0s - 4ms/step - loss: 197.3533 - val_loss: 259.5796
Epoch 10/100
23/23 - 0s - 5ms/step - loss: 192.0785 - val_loss: 249.5500
Epoch 11/100
23/23 - 0s - 5ms/step - loss: 186.5384 - val_loss: 241.0894
Epoch 12/100
23/23 - 0s - 5ms/step - loss: 180.4969 - val_loss: 230.6416
Epoch 13/100
23/23 - 0s - 5ms/step - loss: 175.7516 - val_loss: 220.6802
Epoch 14/100
23/23 - 0s - 5ms/step - loss: 170.5457 - val_loss: 213.218

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 2s - 71ms/step - loss: 4392.3291 - val_loss: 1270.3325
Epoch 2/100
23/23 - 0s - 5ms/step - loss: 1161.8073 - val_loss: 811.4130
Epoch 3/100
23/23 - 0s - 5ms/step - loss: 677.0897 - val_loss: 517.8447
Epoch 4/100
23/23 - 0s - 5ms/step - loss: 472.8434 - val_loss: 394.1813
Epoch 5/100
23/23 - 0s - 5ms/step - loss: 345.0291 - val_loss: 291.1482
Epoch 6/100
23/23 - 0s - 5ms/step - loss: 261.8433 - val_loss: 236.0761
Epoch 7/100
23/23 - 0s - 5ms/step - loss: 218.8482 - val_loss: 207.5067
Epoch 8/100
23/23 - 0s - 5ms/step - loss: 193.6725 - val_loss: 180.9508
Epoch 9/100
23/23 - 0s - 5ms/step - loss: 180.3961 - val_loss: 172.4624
Epoch 10/100
23/23 - 0s - 5ms/step - loss: 171.9115 - val_loss: 172.1031
Epoch 11/100
23/23 - 0s - 5ms/step - loss: 166.2136 - val_loss: 168.8732
Epoch 12/100
23/23 - 0s - 5ms/step - loss: 162.8384 - val_loss: 168.7233
Epoch 13/100
23/23 - 0s - 5ms/step - loss: 159.0569 - val_loss: 167.6505
Epoch 14/100
23/23 - 0s - 5ms/step - loss: 156.0295 - val_loss: 160.

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 2s - 69ms/step - loss: 460.4227 - val_loss: 264.0793
Epoch 2/100
23/23 - 0s - 5ms/step - loss: 301.3285 - val_loss: 254.8239
Epoch 3/100
23/23 - 0s - 5ms/step - loss: 261.3734 - val_loss: 217.6292
Epoch 4/100
23/23 - 0s - 5ms/step - loss: 239.7845 - val_loss: 208.5820
Epoch 5/100
23/23 - 0s - 5ms/step - loss: 227.3573 - val_loss: 192.0834
Epoch 6/100
23/23 - 0s - 4ms/step - loss: 212.1722 - val_loss: 182.0977
Epoch 7/100
23/23 - 0s - 5ms/step - loss: 202.8432 - val_loss: 173.1318
Epoch 8/100
23/23 - 0s - 5ms/step - loss: 192.6421 - val_loss: 164.2699
Epoch 9/100
23/23 - 0s - 5ms/step - loss: 182.8438 - val_loss: 156.6707
Epoch 10/100
23/23 - 0s - 5ms/step - loss: 173.7593 - val_loss: 148.0927
Epoch 11/100
23/23 - 0s - 5ms/step - loss: 169.6243 - val_loss: 156.9543
Epoch 12/100
23/23 - 0s - 5ms/step - loss: 164.5595 - val_loss: 136.3620
Epoch 13/100
23/23 - 0s - 5ms/step - loss: 150.6200 - val_loss: 132.5063
Epoch 14/100
23/23 - 0s - 5ms/step - loss: 144.9250 - val_loss: 132.036

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 2s - 70ms/step - loss: 53135.0781 - val_loss: 16195.5391
Epoch 2/100
23/23 - 0s - 5ms/step - loss: 5437.6455 - val_loss: 898.0249
Epoch 3/100
23/23 - 0s - 5ms/step - loss: 628.7287 - val_loss: 577.8000
Epoch 4/100
23/23 - 0s - 5ms/step - loss: 568.3745 - val_loss: 516.8032
Epoch 5/100
23/23 - 0s - 5ms/step - loss: 495.0279 - val_loss: 501.9994
Epoch 6/100
23/23 - 0s - 5ms/step - loss: 466.4530 - val_loss: 466.5353
Epoch 7/100
23/23 - 0s - 5ms/step - loss: 438.2011 - val_loss: 429.9124
Epoch 8/100
23/23 - 0s - 8ms/step - loss: 410.9825 - val_loss: 411.3816
Epoch 9/100
23/23 - 0s - 5ms/step - loss: 390.0428 - val_loss: 387.3919
Epoch 10/100
23/23 - 0s - 5ms/step - loss: 370.1765 - val_loss: 369.3525
Epoch 11/100
23/23 - 0s - 5ms/step - loss: 351.4210 - val_loss: 347.0151
Epoch 12/100
23/23 - 0s - 5ms/step - loss: 332.9924 - val_loss: 336.2782
Epoch 13/100
23/23 - 0s - 5ms/step - loss: 319.8786 - val_loss: 319.1674
Epoch 14/100
23/23 - 0s - 5ms/step - loss: 308.9926 - val_loss: 30

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 2s - 71ms/step - loss: 3899.6748 - val_loss: 590.1487
Epoch 2/100
23/23 - 0s - 5ms/step - loss: 324.2465 - val_loss: 314.8274
Epoch 3/100
23/23 - 0s - 5ms/step - loss: 244.4622 - val_loss: 236.2523
Epoch 4/100
23/23 - 0s - 5ms/step - loss: 221.8265 - val_loss: 231.7604
Epoch 5/100
23/23 - 0s - 5ms/step - loss: 216.7937 - val_loss: 225.2120
Epoch 6/100
23/23 - 0s - 5ms/step - loss: 213.0031 - val_loss: 221.6033
Epoch 7/100
23/23 - 0s - 5ms/step - loss: 209.3328 - val_loss: 216.3784
Epoch 8/100
23/23 - 0s - 5ms/step - loss: 206.3266 - val_loss: 214.9038
Epoch 9/100
23/23 - 0s - 5ms/step - loss: 203.2754 - val_loss: 207.4488
Epoch 10/100
23/23 - 0s - 5ms/step - loss: 199.7225 - val_loss: 206.4848
Epoch 11/100
23/23 - 0s - 5ms/step - loss: 197.8415 - val_loss: 200.0250
Epoch 12/100
23/23 - 0s - 5ms/step - loss: 194.5353 - val_loss: 200.4438
Epoch 13/100
23/23 - 0s - 5ms/step - loss: 192.5354 - val_loss: 194.6309
Epoch 14/100
23/23 - 0s - 5ms/step - loss: 189.0264 - val_loss: 192.52

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 2s - 71ms/step - loss: 439.9268 - val_loss: 377.9756
Epoch 2/100
23/23 - 0s - 5ms/step - loss: 260.3664 - val_loss: 248.8622
Epoch 3/100
23/23 - 0s - 5ms/step - loss: 205.2437 - val_loss: 205.0337
Epoch 4/100
23/23 - 0s - 5ms/step - loss: 178.9274 - val_loss: 189.2816
Epoch 5/100
23/23 - 0s - 5ms/step - loss: 168.8866 - val_loss: 191.4478
Epoch 6/100
23/23 - 0s - 5ms/step - loss: 157.7443 - val_loss: 159.8186
Epoch 7/100
23/23 - 0s - 5ms/step - loss: 150.0715 - val_loss: 148.4364
Epoch 8/100
23/23 - 0s - 5ms/step - loss: 150.4214 - val_loss: 140.5566
Epoch 9/100
23/23 - 0s - 5ms/step - loss: 138.8018 - val_loss: 147.3304
Epoch 10/100
23/23 - 0s - 5ms/step - loss: 137.6958 - val_loss: 181.5204
Epoch 11/100
23/23 - 0s - 5ms/step - loss: 147.0938 - val_loss: 136.0267
Epoch 12/100
23/23 - 0s - 5ms/step - loss: 131.9213 - val_loss: 130.0400
Epoch 13/100
23/23 - 0s - 5ms/step - loss: 125.8931 - val_loss: 129.1820
Epoch 14/100
23/23 - 0s - 5ms/step - loss: 127.2110 - val_loss: 142.954

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 2s - 87ms/step - loss: 264.3156 - val_loss: 230.9527
Epoch 2/100
23/23 - 0s - 5ms/step - loss: 200.3355 - val_loss: 208.0413
Epoch 3/100
23/23 - 0s - 5ms/step - loss: 193.3133 - val_loss: 235.6855
Epoch 4/100
23/23 - 0s - 5ms/step - loss: 171.2435 - val_loss: 188.5177
Epoch 5/100
23/23 - 0s - 5ms/step - loss: 161.2163 - val_loss: 185.1386
Epoch 6/100
23/23 - 0s - 5ms/step - loss: 156.1108 - val_loss: 175.1844
Epoch 7/100
23/23 - 0s - 5ms/step - loss: 150.4152 - val_loss: 179.5781
Epoch 8/100
23/23 - 0s - 5ms/step - loss: 147.1717 - val_loss: 170.2253
Epoch 9/100
23/23 - 0s - 5ms/step - loss: 143.9910 - val_loss: 165.4017
Epoch 10/100
23/23 - 0s - 5ms/step - loss: 142.9452 - val_loss: 162.6713
Epoch 11/100
23/23 - 0s - 5ms/step - loss: 140.3922 - val_loss: 181.8428
Epoch 12/100
23/23 - 0s - 5ms/step - loss: 140.6787 - val_loss: 156.3001
Epoch 13/100
23/23 - 0s - 5ms/step - loss: 134.3690 - val_loss: 154.5754
Epoch 14/100
23/23 - 0s - 5ms/step - loss: 131.6083 - val_loss: 151.685

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 2s - 71ms/step - loss: 1706.0447 - val_loss: 652.1896
Epoch 2/100
23/23 - 0s - 5ms/step - loss: 537.8658 - val_loss: 462.7278
Epoch 3/100
23/23 - 0s - 5ms/step - loss: 432.8182 - val_loss: 408.1085
Epoch 4/100
23/23 - 0s - 5ms/step - loss: 391.7489 - val_loss: 375.9336
Epoch 5/100
23/23 - 0s - 5ms/step - loss: 360.7369 - val_loss: 347.6374
Epoch 6/100
23/23 - 0s - 5ms/step - loss: 332.9093 - val_loss: 331.3955
Epoch 7/100
23/23 - 0s - 5ms/step - loss: 315.7950 - val_loss: 308.6497
Epoch 8/100
23/23 - 0s - 5ms/step - loss: 307.0688 - val_loss: 295.6089
Epoch 9/100
23/23 - 0s - 5ms/step - loss: 287.5318 - val_loss: 285.9865
Epoch 10/100
23/23 - 0s - 5ms/step - loss: 273.9838 - val_loss: 273.1936
Epoch 11/100
23/23 - 0s - 5ms/step - loss: 267.1912 - val_loss: 268.9678
Epoch 12/100
23/23 - 0s - 5ms/step - loss: 254.0161 - val_loss: 251.4386
Epoch 13/100
23/23 - 0s - 5ms/step - loss: 242.5488 - val_loss: 247.8643
Epoch 14/100
23/23 - 0s - 5ms/step - loss: 235.9522 - val_loss: 244.98

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 2s - 73ms/step - loss: 2062.3831 - val_loss: 1052.3086
Epoch 2/100
23/23 - 0s - 5ms/step - loss: 711.9872 - val_loss: 499.5038
Epoch 3/100
23/23 - 0s - 5ms/step - loss: 418.9000 - val_loss: 350.2165
Epoch 4/100
23/23 - 0s - 5ms/step - loss: 298.9566 - val_loss: 282.5197
Epoch 5/100
23/23 - 0s - 7ms/step - loss: 241.6434 - val_loss: 242.9781
Epoch 6/100
23/23 - 0s - 5ms/step - loss: 212.9720 - val_loss: 219.3014
Epoch 7/100
23/23 - 0s - 5ms/step - loss: 198.4025 - val_loss: 206.0781
Epoch 8/100
23/23 - 0s - 5ms/step - loss: 189.3322 - val_loss: 196.1182
Epoch 9/100
23/23 - 0s - 5ms/step - loss: 183.5140 - val_loss: 189.6192
Epoch 10/100
23/23 - 0s - 5ms/step - loss: 178.2861 - val_loss: 187.2828
Epoch 11/100
23/23 - 0s - 10ms/step - loss: 174.9253 - val_loss: 182.2629
Epoch 12/100
23/23 - 0s - 5ms/step - loss: 171.8939 - val_loss: 179.9522
Epoch 13/100
23/23 - 0s - 5ms/step - loss: 168.5569 - val_loss: 179.5594
Epoch 14/100
23/23 - 0s - 6ms/step - loss: 167.8262 - val_loss: 175.

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 2s - 70ms/step - loss: 22326.1289 - val_loss: 3044.1550
Epoch 2/100
23/23 - 0s - 5ms/step - loss: 964.8981 - val_loss: 556.8462
Epoch 3/100
23/23 - 0s - 5ms/step - loss: 489.9995 - val_loss: 460.8341
Epoch 4/100
23/23 - 0s - 5ms/step - loss: 386.1956 - val_loss: 396.5514
Epoch 5/100
23/23 - 0s - 5ms/step - loss: 344.5566 - val_loss: 365.5414
Epoch 6/100
23/23 - 0s - 5ms/step - loss: 319.3611 - val_loss: 342.8260
Epoch 7/100
23/23 - 0s - 5ms/step - loss: 301.8741 - val_loss: 322.8615
Epoch 8/100
23/23 - 0s - 5ms/step - loss: 286.5810 - val_loss: 303.3297
Epoch 9/100
23/23 - 0s - 5ms/step - loss: 272.2881 - val_loss: 288.4603
Epoch 10/100
23/23 - 0s - 5ms/step - loss: 261.2993 - val_loss: 274.1889
Epoch 11/100
23/23 - 0s - 5ms/step - loss: 250.3348 - val_loss: 261.4187
Epoch 12/100
23/23 - 0s - 5ms/step - loss: 240.8433 - val_loss: 251.1601
Epoch 13/100
23/23 - 0s - 5ms/step - loss: 233.8885 - val_loss: 241.8085
Epoch 14/100
23/23 - 0s - 5ms/step - loss: 226.5068 - val_loss: 234.

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 2s - 71ms/step - loss: 54085.6328 - val_loss: 25070.4668
Epoch 2/100
23/23 - 0s - 5ms/step - loss: 12785.1387 - val_loss: 3237.9858
Epoch 3/100
23/23 - 0s - 5ms/step - loss: 883.4930 - val_loss: 462.6052
Epoch 4/100
23/23 - 0s - 5ms/step - loss: 427.8638 - val_loss: 328.2417
Epoch 5/100
23/23 - 0s - 4ms/step - loss: 321.8492 - val_loss: 320.1080
Epoch 6/100
23/23 - 0s - 5ms/step - loss: 301.0869 - val_loss: 298.0815
Epoch 7/100
23/23 - 0s - 5ms/step - loss: 287.8285 - val_loss: 288.8898
Epoch 8/100
23/23 - 0s - 5ms/step - loss: 276.2627 - val_loss: 273.6475
Epoch 9/100
23/23 - 0s - 5ms/step - loss: 259.6461 - val_loss: 257.7679
Epoch 10/100
23/23 - 0s - 4ms/step - loss: 234.5186 - val_loss: 238.5761
Epoch 11/100
23/23 - 0s - 5ms/step - loss: 221.6066 - val_loss: 239.5119
Epoch 12/100
23/23 - 0s - 5ms/step - loss: 213.8536 - val_loss: 225.9184
Epoch 13/100
23/23 - 0s - 9ms/step - loss: 207.4561 - val_loss: 221.9986
Epoch 14/100
23/23 - 0s - 4ms/step - loss: 201.8954 - val_loss: 

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 2s - 70ms/step - loss: 504.7864 - val_loss: 270.5287
Epoch 2/100
23/23 - 0s - 5ms/step - loss: 286.3010 - val_loss: 245.5359
Epoch 3/100
23/23 - 0s - 5ms/step - loss: 267.5907 - val_loss: 233.3075
Epoch 4/100
23/23 - 0s - 5ms/step - loss: 255.5617 - val_loss: 224.5106
Epoch 5/100
23/23 - 0s - 5ms/step - loss: 242.4118 - val_loss: 212.8860
Epoch 6/100
23/23 - 0s - 4ms/step - loss: 232.4058 - val_loss: 202.3257
Epoch 7/100
23/23 - 0s - 5ms/step - loss: 220.8746 - val_loss: 196.7340
Epoch 8/100
23/23 - 0s - 5ms/step - loss: 212.8158 - val_loss: 184.1615
Epoch 9/100
23/23 - 0s - 5ms/step - loss: 199.8483 - val_loss: 172.3815
Epoch 10/100
23/23 - 0s - 5ms/step - loss: 190.4224 - val_loss: 163.8545
Epoch 11/100
23/23 - 0s - 5ms/step - loss: 183.1753 - val_loss: 156.3464
Epoch 12/100
23/23 - 0s - 5ms/step - loss: 174.9430 - val_loss: 149.8500
Epoch 13/100
23/23 - 0s - 4ms/step - loss: 168.5558 - val_loss: 144.3348
Epoch 14/100
23/23 - 0s - 5ms/step - loss: 163.2676 - val_loss: 141.088

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 2s - 71ms/step - loss: 1196.3098 - val_loss: 733.9888
Epoch 2/100
23/23 - 0s - 5ms/step - loss: 590.4329 - val_loss: 476.9238
Epoch 3/100
23/23 - 0s - 5ms/step - loss: 424.5747 - val_loss: 350.9787
Epoch 4/100
23/23 - 0s - 5ms/step - loss: 342.3292 - val_loss: 296.2038
Epoch 5/100
23/23 - 0s - 5ms/step - loss: 306.3967 - val_loss: 273.7806
Epoch 6/100
23/23 - 0s - 5ms/step - loss: 288.0125 - val_loss: 258.7948
Epoch 7/100
23/23 - 0s - 4ms/step - loss: 278.9743 - val_loss: 248.9388
Epoch 8/100
23/23 - 0s - 4ms/step - loss: 267.8797 - val_loss: 239.0217
Epoch 9/100
23/23 - 0s - 5ms/step - loss: 258.1183 - val_loss: 230.7286
Epoch 10/100
23/23 - 0s - 5ms/step - loss: 246.7762 - val_loss: 233.7619
Epoch 11/100
23/23 - 0s - 5ms/step - loss: 239.2171 - val_loss: 216.0066
Epoch 12/100
23/23 - 0s - 5ms/step - loss: 229.5695 - val_loss: 207.4709
Epoch 13/100
23/23 - 0s - 5ms/step - loss: 221.7083 - val_loss: 201.7193
Epoch 14/100
23/23 - 0s - 5ms/step - loss: 214.5897 - val_loss: 196.89

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 2s - 77ms/step - loss: 10797.1230 - val_loss: 3974.6348
Epoch 2/100
23/23 - 0s - 5ms/step - loss: 1697.7595 - val_loss: 436.0910
Epoch 3/100
23/23 - 0s - 5ms/step - loss: 450.2132 - val_loss: 401.8343
Epoch 4/100
23/23 - 0s - 5ms/step - loss: 396.6537 - val_loss: 371.7101
Epoch 5/100
23/23 - 0s - 5ms/step - loss: 373.9566 - val_loss: 361.2852
Epoch 6/100
23/23 - 0s - 5ms/step - loss: 359.7152 - val_loss: 349.8898
Epoch 7/100
23/23 - 0s - 5ms/step - loss: 342.3455 - val_loss: 339.0675
Epoch 8/100
23/23 - 0s - 5ms/step - loss: 329.5446 - val_loss: 332.2594
Epoch 9/100
23/23 - 0s - 5ms/step - loss: 320.3668 - val_loss: 324.7307
Epoch 10/100
23/23 - 0s - 5ms/step - loss: 311.3742 - val_loss: 317.4401
Epoch 11/100
23/23 - 0s - 5ms/step - loss: 295.4778 - val_loss: 306.3228
Epoch 12/100
23/23 - 0s - 5ms/step - loss: 283.1100 - val_loss: 291.4615
Epoch 13/100
23/23 - 0s - 5ms/step - loss: 272.0469 - val_loss: 280.9882
Epoch 14/100
23/23 - 0s - 5ms/step - loss: 259.6464 - val_loss: 267

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 2s - 71ms/step - loss: 1619.3782 - val_loss: 695.1449
Epoch 2/100
23/23 - 0s - 5ms/step - loss: 634.6433 - val_loss: 532.1368
Epoch 3/100
23/23 - 0s - 5ms/step - loss: 443.8633 - val_loss: 397.4155
Epoch 4/100
23/23 - 0s - 5ms/step - loss: 350.3618 - val_loss: 338.5521
Epoch 5/100
23/23 - 0s - 5ms/step - loss: 295.5711 - val_loss: 280.4958
Epoch 6/100
23/23 - 0s - 5ms/step - loss: 257.8803 - val_loss: 257.1523
Epoch 7/100
23/23 - 0s - 5ms/step - loss: 237.2433 - val_loss: 232.8485
Epoch 8/100
23/23 - 0s - 5ms/step - loss: 221.7827 - val_loss: 221.7403
Epoch 9/100
23/23 - 0s - 5ms/step - loss: 214.1181 - val_loss: 207.3641
Epoch 10/100
23/23 - 0s - 5ms/step - loss: 208.5792 - val_loss: 197.9705
Epoch 11/100
23/23 - 0s - 4ms/step - loss: 199.8020 - val_loss: 192.0366
Epoch 12/100
23/23 - 0s - 5ms/step - loss: 195.0165 - val_loss: 183.0629
Epoch 13/100
23/23 - 0s - 5ms/step - loss: 187.7177 - val_loss: 179.3584
Epoch 14/100
23/23 - 0s - 5ms/step - loss: 184.5440 - val_loss: 171.18

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 2s - 70ms/step - loss: 97053.4688 - val_loss: 40835.6328
Epoch 2/100
23/23 - 0s - 5ms/step - loss: 21621.6973 - val_loss: 7653.8486
Epoch 3/100
23/23 - 0s - 5ms/step - loss: 3978.2014 - val_loss: 1582.1152
Epoch 4/100
23/23 - 0s - 5ms/step - loss: 1424.8392 - val_loss: 1146.3326
Epoch 5/100
23/23 - 0s - 5ms/step - loss: 1199.8914 - val_loss: 1048.7349
Epoch 6/100
23/23 - 0s - 5ms/step - loss: 1083.0818 - val_loss: 852.9677
Epoch 7/100
23/23 - 0s - 5ms/step - loss: 863.2294 - val_loss: 713.4894
Epoch 8/100
23/23 - 0s - 5ms/step - loss: 707.6939 - val_loss: 586.3920
Epoch 9/100
23/23 - 0s - 5ms/step - loss: 608.3834 - val_loss: 523.2787
Epoch 10/100
23/23 - 0s - 5ms/step - loss: 536.6293 - val_loss: 476.0727
Epoch 11/100
23/23 - 0s - 5ms/step - loss: 485.5078 - val_loss: 427.7217
Epoch 12/100
23/23 - 0s - 5ms/step - loss: 445.1880 - val_loss: 395.7000
Epoch 13/100
23/23 - 0s - 4ms/step - loss: 410.8931 - val_loss: 371.2477
Epoch 14/100
23/23 - 0s - 4ms/step - loss: 379.6360 - val

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 2s - 71ms/step - loss: 71164.5312 - val_loss: 28495.0742
Epoch 2/100
23/23 - 0s - 5ms/step - loss: 13689.5020 - val_loss: 3812.1379
Epoch 3/100
23/23 - 0s - 5ms/step - loss: 1686.5172 - val_loss: 699.6572
Epoch 4/100
23/23 - 0s - 5ms/step - loss: 606.5718 - val_loss: 607.7473
Epoch 5/100
23/23 - 0s - 4ms/step - loss: 561.2050 - val_loss: 565.8470
Epoch 6/100
23/23 - 0s - 5ms/step - loss: 522.9601 - val_loss: 526.3087
Epoch 7/100
23/23 - 0s - 5ms/step - loss: 488.7839 - val_loss: 494.1645
Epoch 8/100
23/23 - 0s - 5ms/step - loss: 458.4212 - val_loss: 460.9968
Epoch 9/100
23/23 - 0s - 4ms/step - loss: 428.1772 - val_loss: 431.4177
Epoch 10/100
23/23 - 0s - 5ms/step - loss: 401.7841 - val_loss: 402.0986
Epoch 11/100
23/23 - 0s - 4ms/step - loss: 376.5158 - val_loss: 376.6521
Epoch 12/100
23/23 - 0s - 5ms/step - loss: 354.3159 - val_loss: 351.8076
Epoch 13/100
23/23 - 0s - 5ms/step - loss: 332.3279 - val_loss: 328.5078
Epoch 14/100
23/23 - 0s - 4ms/step - loss: 312.7973 - val_loss:

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 2s - 71ms/step - loss: 341.8888 - val_loss: 208.8530
Epoch 2/100
23/23 - 0s - 5ms/step - loss: 206.0692 - val_loss: 190.1430
Epoch 3/100
23/23 - 0s - 5ms/step - loss: 190.9710 - val_loss: 175.2960
Epoch 4/100
23/23 - 0s - 5ms/step - loss: 173.4808 - val_loss: 160.1873
Epoch 5/100
23/23 - 0s - 5ms/step - loss: 162.3019 - val_loss: 147.6545
Epoch 6/100
23/23 - 0s - 5ms/step - loss: 150.3760 - val_loss: 143.4288
Epoch 7/100
23/23 - 0s - 4ms/step - loss: 141.9700 - val_loss: 136.9948
Epoch 8/100
23/23 - 0s - 5ms/step - loss: 129.1647 - val_loss: 119.6531
Epoch 9/100
23/23 - 0s - 5ms/step - loss: 119.9706 - val_loss: 110.7163
Epoch 10/100
23/23 - 0s - 5ms/step - loss: 114.9281 - val_loss: 106.7787
Epoch 11/100
23/23 - 0s - 5ms/step - loss: 109.5026 - val_loss: 99.4706
Epoch 12/100
23/23 - 0s - 4ms/step - loss: 99.0705 - val_loss: 94.3559
Epoch 13/100
23/23 - 0s - 5ms/step - loss: 94.8446 - val_loss: 91.4757
Epoch 14/100
23/23 - 0s - 5ms/step - loss: 94.8878 - val_loss: 95.9186
Epoch

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 2s - 70ms/step - loss: 592.6017 - val_loss: 348.2024
Epoch 2/100
23/23 - 0s - 5ms/step - loss: 279.4269 - val_loss: 210.3884
Epoch 3/100
23/23 - 0s - 5ms/step - loss: 234.0735 - val_loss: 203.7813
Epoch 4/100
23/23 - 0s - 5ms/step - loss: 225.7890 - val_loss: 198.6839
Epoch 5/100
23/23 - 0s - 5ms/step - loss: 220.8456 - val_loss: 195.0205
Epoch 6/100
23/23 - 0s - 5ms/step - loss: 217.3850 - val_loss: 192.0006
Epoch 7/100
23/23 - 0s - 5ms/step - loss: 213.4596 - val_loss: 189.4516
Epoch 8/100
23/23 - 0s - 5ms/step - loss: 210.8826 - val_loss: 186.7086
Epoch 9/100
23/23 - 0s - 5ms/step - loss: 207.5275 - val_loss: 184.3352
Epoch 10/100
23/23 - 0s - 5ms/step - loss: 205.0512 - val_loss: 182.4268
Epoch 11/100
23/23 - 0s - 5ms/step - loss: 202.3529 - val_loss: 179.9797
Epoch 12/100
23/23 - 0s - 5ms/step - loss: 199.6714 - val_loss: 177.9060
Epoch 13/100
23/23 - 0s - 5ms/step - loss: 197.5500 - val_loss: 175.4277
Epoch 14/100
23/23 - 0s - 5ms/step - loss: 193.8292 - val_loss: 173.151

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 2s - 70ms/step - loss: 476.0859 - val_loss: 266.0009
Epoch 2/100
23/23 - 0s - 5ms/step - loss: 260.2701 - val_loss: 234.0687
Epoch 3/100
23/23 - 0s - 5ms/step - loss: 244.7149 - val_loss: 217.4515
Epoch 4/100
23/23 - 0s - 5ms/step - loss: 228.1272 - val_loss: 196.5174
Epoch 5/100
23/23 - 0s - 5ms/step - loss: 210.4422 - val_loss: 184.0406
Epoch 6/100
23/23 - 0s - 5ms/step - loss: 197.7599 - val_loss: 170.6682
Epoch 7/100
23/23 - 0s - 5ms/step - loss: 188.4031 - val_loss: 162.5251
Epoch 8/100
23/23 - 0s - 5ms/step - loss: 178.1294 - val_loss: 156.9011
Epoch 9/100
23/23 - 0s - 6ms/step - loss: 167.8217 - val_loss: 150.4715
Epoch 10/100
23/23 - 0s - 7ms/step - loss: 157.3638 - val_loss: 140.1785
Epoch 11/100
23/23 - 0s - 7ms/step - loss: 146.3244 - val_loss: 131.8572
Epoch 12/100
23/23 - 0s - 7ms/step - loss: 135.9299 - val_loss: 121.8526
Epoch 13/100
23/23 - 0s - 7ms/step - loss: 122.9364 - val_loss: 112.8201
Epoch 14/100
23/23 - 0s - 7ms/step - loss: 112.8117 - val_loss: 103.295

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 2s - 72ms/step - loss: 392.9948 - val_loss: 410.1440
Epoch 2/100
23/23 - 0s - 5ms/step - loss: 319.2436 - val_loss: 348.3088
Epoch 3/100
23/23 - 0s - 5ms/step - loss: 288.8256 - val_loss: 304.9760
Epoch 4/100
23/23 - 0s - 5ms/step - loss: 262.2719 - val_loss: 282.8605
Epoch 5/100
23/23 - 0s - 5ms/step - loss: 248.7159 - val_loss: 269.9588
Epoch 6/100
23/23 - 0s - 5ms/step - loss: 239.9985 - val_loss: 263.4265
Epoch 7/100
23/23 - 0s - 5ms/step - loss: 231.0708 - val_loss: 246.0749
Epoch 8/100
23/23 - 0s - 5ms/step - loss: 223.1958 - val_loss: 238.8201
Epoch 9/100
23/23 - 0s - 5ms/step - loss: 216.4427 - val_loss: 229.5796
Epoch 10/100
23/23 - 0s - 5ms/step - loss: 209.6758 - val_loss: 224.1536
Epoch 11/100
23/23 - 0s - 6ms/step - loss: 201.2607 - val_loss: 215.1826
Epoch 12/100
23/23 - 0s - 6ms/step - loss: 195.1548 - val_loss: 209.5842
Epoch 13/100
23/23 - 0s - 6ms/step - loss: 187.4828 - val_loss: 202.0448
Epoch 14/100
23/23 - 0s - 5ms/step - loss: 184.3789 - val_loss: 194.948

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 2s - 71ms/step - loss: 1130.0065 - val_loss: 490.5425
Epoch 2/100
23/23 - 0s - 5ms/step - loss: 345.3123 - val_loss: 320.2088
Epoch 3/100
23/23 - 0s - 5ms/step - loss: 306.8098 - val_loss: 302.3840
Epoch 4/100
23/23 - 0s - 5ms/step - loss: 292.8222 - val_loss: 289.5312
Epoch 5/100
23/23 - 0s - 5ms/step - loss: 281.0039 - val_loss: 279.0266
Epoch 6/100
23/23 - 0s - 5ms/step - loss: 272.8148 - val_loss: 269.5037
Epoch 7/100
23/23 - 0s - 5ms/step - loss: 262.1121 - val_loss: 255.7640
Epoch 8/100
23/23 - 0s - 5ms/step - loss: 248.9145 - val_loss: 247.5249
Epoch 9/100
23/23 - 0s - 5ms/step - loss: 237.4071 - val_loss: 234.0323
Epoch 10/100
23/23 - 0s - 5ms/step - loss: 225.6423 - val_loss: 225.9302
Epoch 11/100
23/23 - 0s - 5ms/step - loss: 215.5643 - val_loss: 217.2139
Epoch 12/100
23/23 - 0s - 5ms/step - loss: 200.3517 - val_loss: 221.5338
Epoch 13/100
23/23 - 0s - 8ms/step - loss: 190.4020 - val_loss: 199.6990
Epoch 14/100
23/23 - 0s - 5ms/step - loss: 180.0720 - val_loss: 176.24

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 2s - 102ms/step - loss: 39211.7969 - val_loss: 25494.5996
Epoch 2/100
23/23 - 0s - 5ms/step - loss: 19229.9746 - val_loss: 14298.6768
Epoch 3/100
23/23 - 0s - 5ms/step - loss: 11460.5322 - val_loss: 8903.5986
Epoch 4/100
23/23 - 0s - 5ms/step - loss: 6175.8545 - val_loss: 3925.6433
Epoch 5/100
23/23 - 0s - 5ms/step - loss: 2702.1516 - val_loss: 1925.9264
Epoch 6/100
23/23 - 0s - 5ms/step - loss: 1465.1110 - val_loss: 1075.9260
Epoch 7/100
23/23 - 0s - 5ms/step - loss: 872.4728 - val_loss: 709.0450
Epoch 8/100
23/23 - 0s - 7ms/step - loss: 619.7901 - val_loss: 555.5423
Epoch 9/100
23/23 - 0s - 6ms/step - loss: 483.0600 - val_loss: 445.8397
Epoch 10/100
23/23 - 0s - 5ms/step - loss: 397.4313 - val_loss: 377.6375
Epoch 11/100
23/23 - 0s - 5ms/step - loss: 342.9497 - val_loss: 336.3653
Epoch 12/100
23/23 - 0s - 5ms/step - loss: 311.8532 - val_loss: 311.0174
Epoch 13/100
23/23 - 0s - 5ms/step - loss: 293.3367 - val_loss: 294.4919
Epoch 14/100
23/23 - 0s - 5ms/step - loss: 285.5639 -

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 2s - 70ms/step - loss: 90548.8906 - val_loss: 15206.2861
Epoch 2/100
23/23 - 0s - 5ms/step - loss: 5095.4751 - val_loss: 2266.6567
Epoch 3/100
23/23 - 0s - 5ms/step - loss: 2702.1978 - val_loss: 1839.3704
Epoch 4/100
23/23 - 0s - 5ms/step - loss: 2141.3464 - val_loss: 1670.6265
Epoch 5/100
23/23 - 0s - 5ms/step - loss: 1985.6903 - val_loss: 1554.8323
Epoch 6/100
23/23 - 0s - 5ms/step - loss: 1854.5925 - val_loss: 1457.6752
Epoch 7/100
23/23 - 0s - 5ms/step - loss: 1733.0641 - val_loss: 1366.1213
Epoch 8/100
23/23 - 0s - 5ms/step - loss: 1614.5399 - val_loss: 1280.5808
Epoch 9/100
23/23 - 0s - 9ms/step - loss: 1501.6312 - val_loss: 1196.9008
Epoch 10/100
23/23 - 0s - 5ms/step - loss: 1397.8373 - val_loss: 1121.7084
Epoch 11/100
23/23 - 0s - 5ms/step - loss: 1301.6165 - val_loss: 1050.9418
Epoch 12/100
23/23 - 0s - 5ms/step - loss: 1211.9899 - val_loss: 984.2312
Epoch 13/100
23/23 - 0s - 5ms/step - loss: 1133.8806 - val_loss: 921.0854
Epoch 14/100
23/23 - 0s - 5ms/step - loss: 10

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 2s - 71ms/step - loss: 224168.2031 - val_loss: 108459.6406
Epoch 2/100
23/23 - 0s - 10ms/step - loss: 55762.4961 - val_loss: 21454.1055
Epoch 3/100
23/23 - 0s - 5ms/step - loss: 9662.1279 - val_loss: 2678.8125
Epoch 4/100
23/23 - 0s - 5ms/step - loss: 2496.3608 - val_loss: 2062.9226
Epoch 5/100
23/23 - 0s - 5ms/step - loss: 2019.7456 - val_loss: 1626.8253
Epoch 6/100
23/23 - 0s - 5ms/step - loss: 1687.1119 - val_loss: 1416.8021
Epoch 7/100
23/23 - 0s - 4ms/step - loss: 1468.4753 - val_loss: 1239.5552
Epoch 8/100
23/23 - 0s - 5ms/step - loss: 1287.8586 - val_loss: 1101.0272
Epoch 9/100
23/23 - 0s - 5ms/step - loss: 1143.4224 - val_loss: 985.4458
Epoch 10/100
23/23 - 0s - 4ms/step - loss: 1006.9558 - val_loss: 882.8890
Epoch 11/100
23/23 - 0s - 5ms/step - loss: 898.7950 - val_loss: 801.2519
Epoch 12/100
23/23 - 0s - 5ms/step - loss: 810.2502 - val_loss: 726.9570
Epoch 13/100
23/23 - 0s - 5ms/step - loss: 736.3228 - val_loss: 667.7595
Epoch 14/100
23/23 - 0s - 5ms/step - loss: 676

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 2s - 74ms/step - loss: 1984.4976 - val_loss: 538.5264
Epoch 2/100
23/23 - 0s - 5ms/step - loss: 387.4895 - val_loss: 274.5797
Epoch 3/100
23/23 - 0s - 5ms/step - loss: 270.9505 - val_loss: 219.9252
Epoch 4/100
23/23 - 0s - 5ms/step - loss: 245.9901 - val_loss: 202.0706
Epoch 5/100
23/23 - 0s - 5ms/step - loss: 226.4626 - val_loss: 191.0750
Epoch 6/100
23/23 - 0s - 5ms/step - loss: 217.9155 - val_loss: 187.5905
Epoch 7/100
23/23 - 0s - 5ms/step - loss: 210.3434 - val_loss: 183.4500
Epoch 8/100
23/23 - 0s - 5ms/step - loss: 207.2975 - val_loss: 183.1053
Epoch 9/100
23/23 - 0s - 5ms/step - loss: 205.8564 - val_loss: 178.5303
Epoch 10/100
23/23 - 0s - 5ms/step - loss: 202.4382 - val_loss: 176.0341
Epoch 11/100
23/23 - 0s - 5ms/step - loss: 196.6684 - val_loss: 173.3018
Epoch 12/100
23/23 - 0s - 5ms/step - loss: 191.8411 - val_loss: 170.6403
Epoch 13/100
23/23 - 0s - 5ms/step - loss: 187.8220 - val_loss: 167.9484
Epoch 14/100
23/23 - 0s - 5ms/step - loss: 187.9692 - val_loss: 168.73

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 2s - 69ms/step - loss: 575.7813 - val_loss: 502.4803
Epoch 2/100
23/23 - 0s - 4ms/step - loss: 397.8694 - val_loss: 356.1908
Epoch 3/100
23/23 - 0s - 4ms/step - loss: 305.1516 - val_loss: 270.7478
Epoch 4/100
23/23 - 0s - 4ms/step - loss: 243.0459 - val_loss: 236.3979
Epoch 5/100
23/23 - 0s - 4ms/step - loss: 186.8650 - val_loss: 173.6729
Epoch 6/100
23/23 - 0s - 4ms/step - loss: 156.2464 - val_loss: 151.1067
Epoch 7/100
23/23 - 0s - 4ms/step - loss: 141.8420 - val_loss: 138.5745
Epoch 8/100
23/23 - 0s - 4ms/step - loss: 135.9056 - val_loss: 130.4229
Epoch 9/100
23/23 - 0s - 4ms/step - loss: 126.0357 - val_loss: 130.4584
Epoch 10/100
23/23 - 0s - 4ms/step - loss: 122.3711 - val_loss: 121.3409
Epoch 11/100
23/23 - 0s - 4ms/step - loss: 115.9880 - val_loss: 116.4160
Epoch 12/100
23/23 - 0s - 4ms/step - loss: 114.1957 - val_loss: 114.6484
Epoch 13/100
23/23 - 0s - 4ms/step - loss: 110.8988 - val_loss: 110.7887
Epoch 14/100
23/23 - 0s - 4ms/step - loss: 105.9702 - val_loss: 108.552

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 2s - 68ms/step - loss: 4867.6914 - val_loss: 2003.0404
Epoch 2/100
23/23 - 0s - 4ms/step - loss: 1429.7505 - val_loss: 947.6407
Epoch 3/100
23/23 - 0s - 4ms/step - loss: 708.0736 - val_loss: 506.2915
Epoch 4/100
23/23 - 0s - 4ms/step - loss: 414.9567 - val_loss: 379.7014
Epoch 5/100
23/23 - 0s - 4ms/step - loss: 311.1002 - val_loss: 301.1324
Epoch 6/100
23/23 - 0s - 4ms/step - loss: 239.3429 - val_loss: 259.1770
Epoch 7/100
23/23 - 0s - 4ms/step - loss: 200.3256 - val_loss: 229.0579
Epoch 8/100
23/23 - 0s - 4ms/step - loss: 175.5970 - val_loss: 192.4785
Epoch 9/100
23/23 - 0s - 4ms/step - loss: 156.4201 - val_loss: 172.3387
Epoch 10/100
23/23 - 0s - 4ms/step - loss: 145.6015 - val_loss: 162.3952
Epoch 11/100
23/23 - 0s - 4ms/step - loss: 133.3908 - val_loss: 146.3742
Epoch 12/100
23/23 - 0s - 4ms/step - loss: 126.3751 - val_loss: 138.6387
Epoch 13/100
23/23 - 0s - 4ms/step - loss: 123.2307 - val_loss: 134.8367
Epoch 14/100
23/23 - 0s - 4ms/step - loss: 118.0842 - val_loss: 128.

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 2s - 90ms/step - loss: 10545.7910 - val_loss: 2617.0239
Epoch 2/100
23/23 - 0s - 4ms/step - loss: 962.6587 - val_loss: 297.1042
Epoch 3/100
23/23 - 0s - 4ms/step - loss: 348.8139 - val_loss: 291.5161
Epoch 4/100
23/23 - 0s - 4ms/step - loss: 301.9132 - val_loss: 254.7296
Epoch 5/100
23/23 - 0s - 4ms/step - loss: 270.7823 - val_loss: 234.8159
Epoch 6/100
23/23 - 0s - 4ms/step - loss: 247.0967 - val_loss: 220.6789
Epoch 7/100
23/23 - 0s - 4ms/step - loss: 230.9347 - val_loss: 212.4855
Epoch 8/100
23/23 - 0s - 4ms/step - loss: 219.7527 - val_loss: 208.9447
Epoch 9/100
23/23 - 0s - 4ms/step - loss: 213.8991 - val_loss: 206.8354
Epoch 10/100
23/23 - 0s - 4ms/step - loss: 209.5573 - val_loss: 203.9090
Epoch 11/100
23/23 - 0s - 4ms/step - loss: 205.6214 - val_loss: 202.1678
Epoch 12/100
23/23 - 0s - 4ms/step - loss: 201.5527 - val_loss: 200.0785
Epoch 13/100
23/23 - 0s - 4ms/step - loss: 198.6772 - val_loss: 198.0143
Epoch 14/100
23/23 - 0s - 4ms/step - loss: 196.3901 - val_loss: 196.

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 2s - 71ms/step - loss: 2677.6648 - val_loss: 1217.5081
Epoch 2/100
23/23 - 0s - 5ms/step - loss: 921.1915 - val_loss: 682.6879
Epoch 3/100
23/23 - 0s - 5ms/step - loss: 551.5714 - val_loss: 464.2290
Epoch 4/100
23/23 - 0s - 5ms/step - loss: 397.3945 - val_loss: 356.2650
Epoch 5/100
23/23 - 0s - 5ms/step - loss: 315.2214 - val_loss: 306.2668
Epoch 6/100
23/23 - 0s - 5ms/step - loss: 282.0266 - val_loss: 291.0162
Epoch 7/100
23/23 - 0s - 5ms/step - loss: 273.8855 - val_loss: 285.3073
Epoch 8/100
23/23 - 0s - 5ms/step - loss: 268.8569 - val_loss: 281.7634
Epoch 9/100
23/23 - 0s - 5ms/step - loss: 264.4661 - val_loss: 277.1245
Epoch 10/100
23/23 - 0s - 5ms/step - loss: 259.2552 - val_loss: 271.2365
Epoch 11/100
23/23 - 0s - 5ms/step - loss: 253.9831 - val_loss: 266.4384
Epoch 12/100
23/23 - 0s - 5ms/step - loss: 249.8013 - val_loss: 262.3218
Epoch 13/100
23/23 - 0s - 9ms/step - loss: 246.3190 - val_loss: 259.4218
Epoch 14/100
23/23 - 0s - 5ms/step - loss: 243.9034 - val_loss: 256.7

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 3s - 112ms/step - loss: 1955.9326 - val_loss: 1150.1960
Epoch 2/100
23/23 - 0s - 5ms/step - loss: 821.1157 - val_loss: 716.8466
Epoch 3/100
23/23 - 0s - 5ms/step - loss: 595.5503 - val_loss: 543.9167
Epoch 4/100
23/23 - 0s - 5ms/step - loss: 467.1273 - val_loss: 443.2930
Epoch 5/100
23/23 - 0s - 4ms/step - loss: 397.7598 - val_loss: 397.2953
Epoch 6/100
23/23 - 0s - 5ms/step - loss: 360.5179 - val_loss: 357.3802
Epoch 7/100
23/23 - 0s - 5ms/step - loss: 337.6074 - val_loss: 332.9466
Epoch 8/100
23/23 - 0s - 5ms/step - loss: 313.8293 - val_loss: 313.8881
Epoch 9/100
23/23 - 0s - 5ms/step - loss: 294.0799 - val_loss: 298.9355
Epoch 10/100
23/23 - 0s - 5ms/step - loss: 278.2426 - val_loss: 302.5044
Epoch 11/100
23/23 - 0s - 5ms/step - loss: 272.0107 - val_loss: 277.8137
Epoch 12/100
23/23 - 0s - 4ms/step - loss: 259.8902 - val_loss: 270.6646
Epoch 13/100
23/23 - 0s - 5ms/step - loss: 249.2269 - val_loss: 263.5151
Epoch 14/100
23/23 - 0s - 5ms/step - loss: 245.1267 - val_loss: 256.

C:\Users\SB INFO\anaconda3\Lib\site-packages\keras\src\layers\core\dense.py:87: UserWarning: Do not pass an `input_shape`/`input_dim` argument to a layer. When using Sequential models, prefer using an `Input(shape)` object as the first layer in the model instead.
  super().__init__(activity_regularizer=activity_regularizer, **kwargs)


23/23 - 2s - 70ms/step - loss: 4483.5029 - val_loss: 792.5495
Epoch 2/100
23/23 - 0s - 5ms/step - loss: 651.5499 - val_loss: 400.6123
Epoch 3/100
23/23 - 0s - 5ms/step - loss: 342.6843 - val_loss: 269.5841
Epoch 4/100
23/23 - 0s - 5ms/step - loss: 253.6999 - val_loss: 246.0748
Epoch 5/100
23/23 - 0s - 5ms/step - loss: 229.1357 - val_loss: 239.4521
Epoch 6/100
23/23 - 0s - 5ms/step - loss: 219.2859 - val_loss: 232.9491
Epoch 7/100
23/23 - 0s - 4ms/step - loss: 210.3596 - val_loss: 223.8078
Epoch 8/100
23/23 - 0s - 5ms/step - loss: 201.3326 - val_loss: 218.1891
Epoch 9/100
23/23 - 0s - 5ms/step - loss: 193.0289 - val_loss: 209.4113
Epoch 10/100
23/23 - 0s - 5ms/step - loss: 184.4030 - val_loss: 198.0303
Epoch 11/100
23/23 - 0s - 5ms/step - loss: 176.0858 - val_loss: 183.8006
Epoch 12/100
23/23 - 0s - 5ms/step - loss: 165.8091 - val_loss: 167.4155
Epoch 13/100
23/23 - 0s - 5ms/step - loss: 154.9516 - val_loss: 149.1256
Epoch 14/100
23/23 - 0s - 5ms/step - loss: 142.9982 - val_loss: 139.80

In [17]:
# Compute the mean and standard deviation of the mean squared errors
import numpy as np
mean_mse = np.mean(mean_squared_errors)
std_mse = np.std(mean_squared_errors)

print("Mean of Mean Squared Errors: {}".format(mean_mse))
print("Standard Deviation of Mean Squared Errors: {}".format(std_mse))

Mean of Mean Squared Errors: 105.84191786104583
Standard Deviation of Mean Squared Errors: 184.30329178029154
